qiskit_aer/noise/noise_model.py

```diff
@@ -458,7 +435,7 @@ def from_backend(
     @classmethod
     def from_backend_properties(
         cls,
-        backend_properties: BackendProperties,
+        backend_properties: "BackendProperties",
```

Comment:
I'd mention below in the argument list that `BackendProperties` is assumed to be like the one in the `qiskit_ibm_runtime.models.backend_properties` module